test/evergreen.yml

```diff
@@ -4223,6 +4250,53 @@ tasks:
   - <<: *recovery-stress-test
     name: recovery-stress-test-3
     tags: ["stress-test-3", "stress-test-zseries-3"]
+   # FIXME-WT-14566: Enable disagg leader mode on all platforms once failures has been resolved.
+  - <<: *format-stress-test-disagg-leader
+    name: format-stress-test-disagg-leader-1
+    tags: ["stress-test-disagg-fail"]
+  - <<: *format-stress-test-disagg-leader
+    name: format-stress-test-disagg-leader-2
+    tags: ["stress-test-disagg-fail"]
+  - <<: *format-stress-test-disagg-leader
+    name: format-stress-test-disagg-leader-3
+    tags: ["stress-test-disagg-fail"]
+  - <<: *format-stress-test-disagg-leader
+    name: format-stress-test-disagg-leader-4
+    tags: ["stress-test-disagg-fail"]
+  - <<: *format-stress-test-disagg-leader
+    name: format-stress-test-disagg-leader-5
+    tags: ["stress-test-disagg-fail"]
```

Comment:
Do we really need to run format stress test 5 times at the initial stage? The cycles may not be very effective capturing novel issues 5 hrs vs 1 hr.

Reply:
Sorry, I overlooked the 3 mins timer setting and the fact of overall 15 mins testing turnaround. All good.